wagtail/admin/templates/wagtailadmin/shared/side_panels/includes/action_list_item.html

```diff
@@ -2,33 +2,31 @@
 <div class="w-py-6 w-space-y-3">
 
     {% block content %}
-        <section class="w-flex w-space-x-3" aria-labelledby="status-sidebar-{{ title|cautious_slugify }}">
-            {% icon name=icon_name class_name='w-w-5 w-h-5 w-text-primary w-flex-shrink-0' %}
+    <section class="w-flex w-space-x-3" aria-labelledby="status-sidebar-{{ title|cautious_slugify }}">
```

Comment:
Just a reminder - try to avoid broad nesting changes of templates unless critical to the changes being made to fix the issue. 

It's usually it necessary and makes it harder to review the PR.